docs/pages/server-access/guides/openssh.mdx

```diff
@@ -562,6 +562,27 @@ $ ssh -F ssh_config_teleport ${USER?}@node2.leafcluster.${CLUSTER}
 
 ## Revoke an SSH certificate
 
-To revoke the current Teleport CA and generate a new one, run `tctl auth rotate`. Unless you've highly automated your
-infrastructure, we would suggest you proceed with caution as this will invalidate the OpenSSH
-and Host CAs, meaning that the new CAs will need to be exported to every OpenSSH-based machine again using `tctl auth export` as above.
+Teleport's approach to using short-lived certificates for all infrastructure
+access means that it can generate large numbers of certificates every day. For
+this reason, Teleport does not support traditional certificate revocation.
+
+There are two options available for revocating access: CA rotations, and Teleport locks.
+
+### CA Rotations
+
+To generate a new certificate authority and invalidate user certificates issued
+by the current CA, run `tctl auth rotate --type-user`. This can be a disruptive
```

Comment:
I recommend putting the "This can be a disruptive change..." sentence after "This process will require..." since the latter sentence clarifies what "lack automation" means.